docs/source/examples/pennylane-ibmq-backends.md

```diff
@@ -61,12 +61,14 @@ from qiskit_ibm_provider import IBMProvider
 
 USE_REAL_HARDWARE = False
 
+# As of 14 April 2024, PennyLane is not compatible with Qiskit 1.0,
+# but PennyLane plans to support the upgrade, soon
```

Comment:
Should we add a `TODO` or something similar here so we can find it easily later? Otherwise I fear this comment will live on indefinitely 😛.

Reply:
TODO added